Gemfile

```diff
@@ -105,7 +105,7 @@ group :development, :test do
   gem "spring-commands-rspec"
   gem "teaspoon-jasmine"
   gem "test-unit", "~> 3.0"
-  gem "thin"
+  gem "thin", "~> 1.7.2"
```

Comment:
I usually only like using it where we're worried about breaking changes (so things like devise, simple_form) on minor versions, where we wouldn't want to go from say 1.4.2 to 1.5.0 withouts some testing and validating the newer version works.

In general, I don't like to lock down things in the development group because those should theoretically be able to upgrade as much as they want and break very little, and at the very least will never break across production. 

But you're right in that it sometimes does help us make sure we always increase the versions in the downstreams. If we're worried about making sure we're at a minimum version, I would consider using `>= 1.7.2`, which will also allow going up to 1.8 if that gets released.